target/linux/ar71xx/base-files/etc/board.d/02_network

```diff
@@ -424,7 +424,8 @@ ar71xx_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0@eth0" "1:lan:1" "2:lan:2" "3:lan:3" "4:lan:4"
 		;;
-	tl-wr842n-v2)
+	tl-wr842n-v2|\
+	tl-wr942n-v1)
 		ucidef_set_interfaces_lan_wan "eth1.1" "eth0"
 		ucidef_add_switch "switch0" \
 			"0@eth1" "1:lan:4" "2:lan:1" "3:lan:2" "4:lan:3"
```

Comment:
@pepe2k I cannot test this easily with my hardware, but there are ways to free GPIO pins used by UART in older Atheros chips https://wiki.openwrt.org/doc/recipes/turnoff.uart.to.gpio 
Also it seems it is possible to free JTAG pins https://wiki.openwrt.org/doc/recipes/turnoff.jtag.to.gpio
I don't know if that method works with the newer one here, but it's worth trying.

If that works, it's not hard to clone **package/base-files/files/lib/preinit/30_failsafe_wait** and make a script that does what I said above.